TodoList/Src/TodoList.Contracts/DI/ContainerLifetimeEnum.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace TodoList.Contracts.DI
+{
+    public enum ContainerLifetimeEnum
```

Comment:
`"enum"` should not be a part of the name. You can name it as (and see [next comment](https://github.com/JarekParal/kentico-onboarding-cs/pull/4/files#diff-e432ac733331690d87e287ce812a2369R10))
```suggestion
    public enum Lifetime
```

Reply:
1afc897